### ISSUE TITLE
failure output and test names are not sequenced correctly

### ISSUE THREAD

**Issue by simonmichael** [joelburget/easytest#8]
expectEq's failure message can show up in the wrong place, eg:
```
OK 1
OK 2
5 FAILURE expected to be equal: ...
  ...
OK 3
OK 4
FAILED 5
OK 6
```

**Comment by joelburget** [comment]
I've noticed this as well. I'm already working on a change I hope will address this.

**Comment by joelburget** [comment]
This should be fixed by the 0.3 release.
